qatrack/qa/views/review.py

```
@@ -733,4 +731,7 @@ def get_page_title(self):
 
     def get_queryset(self):
         qs = super(UTCInstances, self).get_queryset()
-        return qs.filter(unit_test_collection__pk=self.kwargs["pk"])
+        try:
+            return qs.filter(unit_test_collection__pk=self.kwargs["pk"])
+        except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)